internal/cli/atlas/deployments/setup.go

```diff
@@ -221,10 +221,6 @@ func (opts *SetupOpts) planSteps(ctx context.Context) (steps int, needPodmanSetu
 }
 
 func (opts *SetupOpts) createLocalDeployment(ctx context.Context) error {
-	if err := podman.Installed(); err != nil {
-		return err
-	}
-
```

Comment:
why did we remove this? Is this run as part of `opts.PodmanClient.Ready(ctx)` inside `LocalDeploymentPreRun`?

Reply:
correct!